shopinvader/views/sale_view.xml

```diff
@@ -86,4 +86,16 @@
         </field>
     </record>
 
+    <record id="sale.action_quotations" model="ir.actions.act_window">
+        <field
+            name="domain"
+        >['&amp;', ('state', 'not in', ('draft', 'sent', 'cancel')), ('typology', '=', 'sale')]</field>
```

Comment:
I think we have two issue here.
First your domain is wrong as you hide all quotation.
Secondly, in native there is not domain on quotation (you can see in sale module)

So we should only have for domain [('typology', '=', 'sale')]


Then in our usability module we can have a glue module for compatibility with sale-usability that will add the domain for showing only quotation and hiding cart.

It's a pity that there is not way to inherit such domain.

Note : an alternative to avoid creating a glue module will be in the sale-usability module to inherit on python side to modify the domain and add the extra domain state

Reply:
oops, bad copy paste
updated

Reply:
I think this answers the need
https://github.com/OCA/server-tools/tree/14.0/base_view_inheritance_extension
but it's another dependency
what do you think for sale_usability ?

Reply:
https://github.com/akretion/odoo-usability/pull/159